projects/canopy/src/lib/card/card.notes.ts

```diff
@@ -30,28 +30,86 @@ or for a Product card:
 
 ~~~html
 <lg-card>
-  <lg-card-header>
-    <lg-card-title headingLevel="3">
-      Card Title
-    </lg-card-title>
-    <lg-card-subtitle>
-      Payroll Reference Number P23456
-    </lg-card-subtitle>
-    <lg-card-principle-data-point>
-      <lg-card-principle-data-point-label>
-        Last payment (after tax and deductions)
-      </lg-card-principle-data-point-label>
-      <lg-card-principle-data-point-value>
-        <span><span class="lg-font-size-3">£</span>230.20</span>
-      </lg-card-principle-data-point-value>
-      <lg-card-principle-data-point-date>
-        as of 01 Jan 2020
-      </lg-card-principle-data-point-date>
-    </lg-card-principle-data-point>
-  </lg-card-header>
+  <lg-card-content>
+    <div lgRow>
+      <div lgCol=12 lgColMd=6>
```

Comment:
Just a small one but the columns values should have `"` e.g.:
```
<div lgCol="12" lgColMd="6">
```

There are few instances of this in here.

Reply:
Done.